src/index.html

```diff
@@ -250,29 +250,29 @@ <h2 class="section-title">Contacts us</h2>
                   >
                 <textarea
                   class="info__input info__input-textarea"
-                  placeholder="Message"
-                >
-                </textarea>
-
-                <a
-                  class="info__button"
-                  href="#"
+                  placeholder="Message"></textarea>
+                <input
```

Comment:
It should be `submit` button to validate fields on click as well (all should be required with the correct type - email for email field). Read [the checklist](https://github.com/mate-academy/layout_landing-page/blob/main/checklist.md) to figure out how to do it correctly